LayoutTests/http/wpt/mediarecorder/MediaRecorder-audio-samplingrate-change.html

```diff
@@ -0,0 +1,66 @@
+<!doctype html>
+<html>
+<head>
+    <title>MediaRecorder with sampling rate change</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<script>
+    async function record(stream, mimeType)
+    {
+        await stream.getAudioTracks()[0].applyConstraints({ sampleRate:48000 });
+        const recorder = new MediaRecorder(stream, { mimeType : mimeType });
+        const promise = new Promise((resolve, reject) => {
+            recorder.ondataavailable = (e) => resolve(e.data);
+            setTimeout(reject, 5000);
+        });
+        recorder.start();
+        setTimeout(() => recorder.stop(), 2500);
+        return promise;
+    }
+
+    promise_test(async (t) => {
+        const stream = await navigator.mediaDevices.getUserMedia({ audio : true });
+
+        setTimeout(() => {
+            stream.getAudioTracks()[0].applyConstraints({sampleRate:44100});
+        }, 1000);
```

Comment:
This makes it a really long test. Does it have to go for a whole second before changing?

Reply:
MediaStream may take a while to start. I found that with a lower value we may not get anything. Most of the MediaRecorder opt test are over 2.5s each !

Reply:
I'll lower it to 0.5s